<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="TestdataLabel">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>40</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Testdata</string>
    </property>
   </widget>
   <widget class="QLabel" name="SubmitdataLabel">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>80</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Submitdata</string>
    </property>
   </widget>
   <widget class="QLabel" name="SpendtimeLabel">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>120</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Spend time</string>
    </property>
   </widget>
   <widget class="QLabel" name="Correctlabel">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>160</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Correct or not</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="TestdataLine">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>40</y>
      <width>500</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="SubmitdataLine">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>80</y>
      <width>500</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="SpendtimeLine">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>120</y>
      <width>500</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="CorrectLine">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>160</y>
      <width>500</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="Slectfloorbox">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>250</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>7</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>8</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>9</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>11</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>12</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>13</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>14</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>15</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>16</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>17</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>18</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>19</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>20</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>21</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>22</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>23</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>24</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>25</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>26</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>27</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>28</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>29</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>30</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="RunButton">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>250</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Run</string>
    </property>
   </widget>
   <widget class="QPushButton" name="StartSimulationButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>200</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Simulation</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PeopleInformationButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>320</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>People Information</string>
    </property>
   </widget>
   <widget class="QLabel" name="FinishLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>190</y>
      <width>200</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Finish!!</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="DestinationLine">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>420</y>
      <width>500</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="PeopleNumLine">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>380</y>
      <width>500</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="DestinationLabel">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>420</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Destination</string>
    </property>
   </widget>
   <widget class="QLabel" name="PeopleNumLabel">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>380</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>People Num</string>
    </property>
   </widget>
   <widget class="QLabel" name="ScoreLabel">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>200</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Score</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ScoreLine">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>200</y>
      <width>500</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="NowElevatorLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>120</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Now Elevator</string>
    </property>
   </widget>
   <widget class="QLabel" name="PeopleInElevatorLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>400</y>
      <width>120</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Peopel in Elevator</string>
    </property>
   </widget>
   <widget class="QLabel" name="DistanceLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>450</y>
      <width>120</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Distance</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="NowElevatorLCD">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>350</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="PeopleInElevatorLCD">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>400</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="DistanceLCD">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>450</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
